angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -50,10 +50,10 @@
         <div class="sub-container">
             <div class="container-label">Dialog</div>
             <nimble-dialog #dialog>
-                <h2>This is a dialog</h2>
-                <p>It opened when you pushed the button</p>
-                <nimble-button (click)="closeDialog('cancel pressed')">Cancel</nimble-button>
-                <nimble-button (click)="closeDialog('OK pressed')">OK</nimble-button>
+                <div slot="title">This is a dialog</div>
+                <div>It opened when you pushed the button</div>
```

Comment:
Should double check examples / tests, etc to make sure they use the span recommendation